pyproject.toml

```diff
@@ -33,7 +33,7 @@ requires-python = ">=3.8"
 dependencies = [
   "click",
   "ixmp >= 3.8.0",
-  "genno[pyam] >= 1.20",
+  "genno[pyam] >= 1.20, < 1.23",
```

Comment:
In the past I've done this in the workflow file, so the commit history of pyproject.toml doesn't contain changes that won't be released. See for instance https://github.com/iiasa/message_ix/commit/76bdd30297d5e5eff31a5ac5f7b42c048a4e63b2

I guess this way is also fine, but it does need a comment (like the one in the above example) to indicate it's a temporary work-around, what relevant issue is, and that it should be removed before a release.

Reply:
I've dropped the commit changing pyproject.toml and instead included one that install `genno < 1.23` in the pytest CI workflow. The nightly workflow seems to work just fine with `genno == 1.23`.